frontend/src/Pages/Student/Course/EnrolledCourses.jsx

```
@@ -0,0 +1,169 @@
+            {/* Semester Tabs */}
+            <div className="flex space-x-4 border-b pb-2">
+                {allSemesters.map((semester) => (
+                    <button
+                        key={semester}
+                        className={`px-4 py-2 rounded-t-md transition-all ${
+                            activeSemester === semester
+                                ? "bg-gray-300 text-gray-800 font-semibold"
+                                : "text-gray-300 hover:bg-gray-800"
+                        }`}
+                        onClick={() => setActiveSemester(semester)}
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)